### ISSUE TITLE
Add Time to Datepickers in mytable.grid

### ISSUE THREAD

**Issue by Isofff** [volkanceylan/Serenity#1074]
Sorry if this was answered before, but I can't find the exact answer searching in issues with my average developer/english skills.

Is there a simple way to add time to the DatePickers Serenity use to filter Date/DateTime attributes ?

![sin titulo](https://cloud.githubusercontent.com/assets/20929712/17756091/614d9314-64a3-11e6-93c3-3f35775005b8.png)

I tried to add jQuery DateTimePicker using the Serenity-guide in "How To: Use a Third Party Plugin With
Serenity", but I didn't have success with it.

How can I achieve this?

Thanks in advance for the answer and your excelent work.

**Comment by Isofff** [comment]
What I tried was adding bootstrap-datetimepicker.js and bootstrap-datetimepicker.min to my Scripts folder.
Then I added the .css's to my content folder.

Next I added these lines to _LayoutHead:
@Html.Script("~/Scripts/bootstrap-datetimepicker.js")
@Html.Stylesheet("~/Content/bootstrap-datetimepicker.css")

But I can't understand the part of creating the editor or how to use the new datetimepicker in my field.
